components/Dialogs/src/MDCAlertController.m

```diff
@@ -687,7 +725,11 @@ - (void)setupAlertView {
     self.alertView.titleLabel.adjustsFontForContentSizeCategory =
         self.adjustsFontForContentSizeCategory;
   }
-  self.alertView.messageLabel.text = self.message;
+  if (self.attributedMessage.count > 0) {
+    self.alertView.messageLabel.attributedText = self.attributedMessage;
```

Comment:
Using Apple's default API for attributed text for UILabel, what is the plan for supporting dynamic fonts?

Reply:
Just tested this and it looks like attributed strings work as expected:

```swift
let label = UILabel()
label.attributedText = NSAttributedString(string: "Test text")
label.frame = CGRect(x: 40, y: 100, width: 100, height: 100)
label.font = UIFont.preferredFont(forTextStyle: .body)
label.adjustsFontForContentSizeCategory = true
view.addSubview(label)
```

Reply:
Have you also verified a scenario of Attributed Text with links? Can you verify that links are tappable?

Reply:
I have not, but my understanding is that this API would not support reasonably links out of the box.

Reply:
Should this support support be provided? I believe it was the reason why original CL from me proposing quite identical change was declined by galiak@

Reply:
The original CL was declined because this behavior can be implemented using the accessoryView as well and the original CL you’d sent out hadn’t yet demonstrated the full complexity of the implementation, for which your intended use case of supporting links will be much greater than the intended use case here.

We’re still going to push back on support for links here if it means significant additional complexity, in which case accessoryView is still the recommended solution. If there’s a simple way to support links without substantially increasing complexity though then we can def discuss more.

Reply:
Unfortunately 'accessoryView' solution is not sufficient for our team use case :( We can discuss it internally though. For this PR, I think it's a great first step towards full support for attributed string, but we have to make it very clear in the documentation of `attributedMessage` property, that links are not supported. Can we please add a disclaimer?

Reply:
+1

Reply:
> This looks good to me. I'm inclined to approve.
> Just out of curiosity, @blastervla, can you elaborate a little on this sentence - "... to provide further customizability and allow for a more liberal use of the component"? What use-cases or customizability this PR will enable for you?

Hi @galiak11! Of course!

Long story short, I'm currently developing a tabletop RPG companion app, and I'd like to make use of the `MDCAlertController` to display what new features the player earned when leveling up a character. As most of the times these are modifications of a currently owned feature, I'm using a simple diff algorithm to calculate the difference and highlight the new parts in green color 😁